plugins/chanbackup.c

```diff
@@ -404,15 +779,37 @@ static const struct plugin_command commands[] = { {
 		"Populates the DB with stub channels",
 		"returns stub channel-id's on completion",
 		json_emergencyrecover,
-	}
+	},
+        {
+                "peerstoragebkp",
+                "recovery",
+                "Grabs the scb from datastore (if exists)",
```

Comment:
Could be more descriptive of what this does?

```
Checks if i have got a backup from a peer, and if so, will stub those channels in the database and if is successful, will return list of channels that have been successfully stubbed
```

Reply:
Okay, these are some really nice questions:
Ans. 1) This would be used in case of data loss. The user could connect to their peers and collect their storage.

Ans. 2) Hmm, That's an excellent suggestion, everytime we receive scb we can automatically stub the channels in the DB (which are not already into it).

Ans. 3) This feature would be used as a last resort to recover the funds in case of severe data loss, This can only help users to recover funds in case of severe data loss, This would never harm existing channels and funds.

Ans. 4) We skip over them and do not stub them in the DB.